src/components/ciPipeline/CiPipelineSourceConfig.tsx

```diff
@@ -134,8 +137,13 @@ export function CiPipelineSourceConfig({
                             <div>
                                 {!baseText && (
                                     <div className="flex left">
-                                        <div className="dc__ellipsis-right">{sourceValueBase}</div>
-                                        <Info className="icon-dim-12 fcn-5 ml-4" />
+                                        <div
+                                            className="dc__ellipsis-right"
```

Comment:
we can use cn-5 instead of style based on condition

Reply:
didn't understand, will connect for more clarity

Reply:
done